packages/mui-material-next/src/InputBase/InputBase.test.tsx

```diff
@@ -228,42 +238,56 @@ describe('<InputBase />', () => {
       const { getByTestId } = render(
         <InputBase
           inputComponent="span"
-          slotProps={{ input: { 'data-testid': 'input-component' } }}
+          slotProps={{
+            input: { 'data-testid': 'input-component' } as InputBaseInputSlotPropsOverrides,
+          }}
         />,
       );
       expect(getByTestId('input-component')).to.have.property('nodeName', 'SPAN');
     });
 
     it('should inject onBlur and onFocus', () => {
-      let injectedProps;
-      const MyInputBase = React.forwardRef(function MyInputBase(props, ref) {
+      let injectedProps: any = {};
+      const MyInputBase = React.forwardRef(function MyInputBase(
+        props: { ownerState: InputBaseOwnerState } & Record<string, unknown>,
+        ref: React.ForwardedRef<HTMLInputElement>,
+      ) {
         injectedProps = props;
         const { ownerState, ...other } = props;
         return <input ref={ref} {...other} />;
       });
 
       render(<InputBase inputComponent={MyInputBase} />);
 
-      expect(typeof injectedProps.onBlur).to.equal('function');
-      expect(typeof injectedProps.onFocus).to.equal('function');
+      if (injectedProps) {
```

Comment:
When would this be undefined? Are we accidentally skipping these expect statements?

Reply:
@mnajdova Fixed ~ it's not needed, I accidentally left it there while trying to fix some TS issue!

Reply:
Perfect, thanks!